### PR TITLE
Feature Request: Add copy-to-clipboard & file write features

### PR DESCRIPTION
**Pull request by yashksaini-coder**
Task:
Integrate the copy-to-clipboard and file write functionalities into the [scrap.py]

✅ Fixes: #141 

@this-is-yaash can you please review this PR, and also assign it to me with proper labels

### CODE REVIEW

File: package/scrap/grab.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            html_content = response.read().decode("utf-8")
+        match = re.search(
+            r'<textarea[^>]*id="cl1pTextArea"[^>]*>(.*?)</textarea>',
+            html_content,
+            re.DOTALL,
+        )
+        if match:
+            content = html.unescape(match.group(1)).strip()
+            if content:
+                return None
```
Comment (this-is-yaash) [inline package/scrap/grab.py:49]:
```suggestion
                return content
```

**Review by this-is-yaash — COMMENTED** [review]
good work, your implementation is just like what I've expected!,🎉

We can merge it right away, but....

It fetches the plain text and didn't have any indentations. Please make sure this fetches the text with indentation and we can merge it ASAP!

**Comment by yashksaini-coder** [comment]
> good work, your implementation is just like what I've expected!,🎉
> 
> We can merge it right away, but....
> 
> It fetches the plain text and didn't have any indentations. Please make sure this fetches the text with indentation and we can merge it ASAP!

Also can you co-author this PR

**Comment by this-is-yaash** [comment]
Increasing level for writing test cases, and ease of execution of program.
@yashksaini-coder 
merging it!

**Pull request merged**